src/Zobrist.cpp

```diff
@@ -32,6 +33,11 @@ void Zobrist::init_zobrist(Random& rng) {
         }
     }
 
+    for (int j = 0; j < FastBoard::MAXSQ; j++) {
+        Zobrist::zobrist_ko[j]  = ((std::uint64_t)rng.randuint32()) << 32;
+        Zobrist::zobrist_ko[j] ^= (std::uint64_t)rng.randuint32();
```

Comment:
At some point Random() should expose randuint64(no arg) and this can all be cleaned up using std::for_each or whatever.